src/routes/properties/propertiesRoute.ts

```diff
@@ -50,6 +60,11 @@ function toResponse(property: Property, toUrl: (key: string) => string): Propert
     landmarks: property.landmarks,
     opportunities: property.opportunities,
     price: property.price,
+    ratings: property.ratings,
+    rating: property.ratings.length
```

Comment:
let's create a function which takes in a property and returns the rating - this is a business operation, so we need extra granularity

Reply:
ok